pkg/manager/member/tikv_member_manager.go

```diff
@@ -372,6 +374,28 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 		}
 	}
 
+	var additionalVolumeClaims []corev1.PersistentVolumeClaim
+	if len(tc.Spec.TiKV.StorageVolumes) > 0 {
+		for _, storageVolume := range tc.Spec.TiKV.StorageVolumes {
+			storageRequest, err := controller.ParseStorageRequest(corev1.ResourceList{
+				corev1.ResourceStorage: resource.MustParse(storageVolume.StorageSize),
+			})
+			if err != nil {
+				return nil, fmt.Errorf("cannot parse storage request for tikv, tidbcluster %s/%s, error: %v", tc.Namespace, tc.Name, err)
```

Comment:
We should validate this field first.
And if we still hit an error here, is it better to print the error log and continue to sync others instead of returning the error here?